recipes/glimpse-bio/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Build the binaries
+make \
+	-j 4 \
+	CC="$CC -std=c++11" \
+	DYN_LIBS="-lz -lpthread -lbz2 -llzma -lcurl -lhts -ldeflate" \
+	CXXFLAG="$CXXFLAGS ${PREFIX}/lib" \
+	LDFLAG="$LDFLAGS" \
```

Comment:
^ the makefiles from GLIMPSE are non-standard, reported to the GLIMPSE developers: https://github.com/odelaneau/GLIMPSE/pull/64